### PR TITLE
Fixed systemtests

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Bugfix
- Refactoring

### Description
Fixed system tests

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests testcase=shared.standard.web.FirefoxConsoleTest

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Comment by kornys** [comment]
obviously build failed on checkstype in GH actions too, so it does not make any sense to restart jenkins, because it will fail on same error

**Comment by ajaypratap003** [comment]
> obviously build failed on checkstype in GH actions too, so it does not make any sense to restart jenkins, because it will fail on same error

@kornys , could you please take a look on this build.

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests profile=shared testcase=*ConsoleTest

**Comment by kornys** [comment]
Again could you please tell me why do you still restart build? It will always fail until someone make a fix for checkstyle.

**Comment by kornys** [comment]
As we said with Ulf check checkstyle.xml output, you will see why checkstyle fails

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java [hunk 1]
```diff
@@ -707,7 +707,7 @@ public void addFilter(FilterType filterType, String filterValue) throws Exceptio
                 selenium.clickOnItem(getStatusFilterDropDownItem());
                 selenium.clickOnItem(getSelectStatusDropDown(), "Status phase dropdown");
                 selenium.clickOnItem(getSelectStatusDropDown()
-                        .findElement(By.id("al-filter-select-status-dropdown-itemstatus" + filterValue.substring(0, 1).toUpperCase() + filterValue.substring(1))));
+                        .findElement(By.id("al-filter-select-status-dropdown-item" + filterValue.toLowerCase()})));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java:710]: @ajaypratap003 obviously on this line is and issue `}` char brake compilation... You should try compile your code before pushing.

Comment (ajaypratap003) [reply systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java:710]: thanks @kornys for notice this typo issue. I"ll fix this.

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=*ConsoleTest

**Pull request merged**